dev_settings.py

```diff
@@ -75,8 +75,7 @@
 CSRF_SOFT_MODE = False
 
 # Make a dir to use for storing attachments as blobs on the filesystem
-shared_dirname = os.path.join(os.path.dirname(os.path.realpath(__file__)),
-                              'sharedfiles')
+shared_dirname = '/mnt/sharedfiles'
```

Comment:
The `dockersettings.py` file already defines the `SHARED_DRIVE_ROOT` settings as `/sharedfiles`